server/Arcadia.Assistant.Web/Controllers/PendingRequestsController.cs

```diff
@@ -6,19 +6,20 @@
 
     using Akka.Actor;
 
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+
     using Arcadia.Assistant.Server.Interop;
+    using Arcadia.Assistant.Web.Authorization;
     using Arcadia.Assistant.Web.Configuration;
     using Arcadia.Assistant.Web.Employees;
     using Arcadia.Assistant.Web.Models.Calendar;
     using Arcadia.Assistant.Web.Users;
-
-    using Microsoft.AspNetCore.Authorization;
-    using Microsoft.AspNetCore.Http;
-    using Microsoft.AspNetCore.Mvc;
-    using UserPreferences;
+    using Arcadia.Assistant.Web.UserPreferences;
 
     [Route("api/pending-requests")]
-    [Authorize]
+    [Authorize(Policies.UserIsEmployee)]
```

Comment:
in my defense, it didn't harm :P

Reply:
Yes, it is used by default, if no one is set. But explicit requirement is more clear.